pkg/collector/corechecks/servicediscovery/module/impl_linux.go

```diff
@@ -260,19 +266,29 @@ func newNetIPSocket(file string, expectedState uint64) (map[uint64]uint16, error
 // protocols for the provided namespace. Based on snapshotBoundSockets() in
 // pkg/security/security_profile/activity_tree/process_node_snapshot.go.
 func getNsInfo(pid int) (*namespaceInfo, error) {
-	tcp, err := newNetIPSocket(kernel.HostProc(fmt.Sprintf("%d/net/tcp", pid)), tcpListen)
+	tcp, err := newNetIPSocket(kernel.HostProc(fmt.Sprintf("%d/net/tcp", pid)), tcpListen, nil)
```

Comment:
nit: instead of nil, declare `var noIgnore = nil` with a simple documentation
it can help readability

Reply:
Done